### PR TITLE
Test/stark

### PR DESCRIPTION
**Pull request by Okm165**

### CODE REVIEW

**Comment by Okm165** [comment]
DO NOT MERGE TO MAIN scarb test takes 40G of ram due to stark_verify test

**Comment by Okm165** [comment]
all tests are WORKING

**Comment by Okm165** [comment]
major improvement in resources consumption thanks to [9b7a354](https://github.com/HerodotusDev/cairo-verifier/pull/85/commits/9b7a35455e93a0c346e1c0cf45b9ff0a1344507c)

**Comment by neotheprogramist** [comment]
This PR fixes https://github.com/HerodotusDev/cairo-verifier/pull/87

File: src/stark.cairo [hunk 1]
```diff
@@ -1,15 +1,19 @@
+use core::debug::PrintTrait;
```
Comment (neotheprogramist) [inline src/stark.cairo:1]: Can be removed

File: src/stark.cairo [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+use core::option::OptionTrait;
+use core::traits::TryInto;
```
Comment (neotheprogramist) [inline src/stark.cairo:3]: I think this also may be removed

**Review by neotheprogramist — APPROVED** [review]
LGTM

**Pull request merged**